src/game/GameScenario.ts

```diff
@@ -51,26 +43,35 @@ export class BasicGameScenario implements GameScenario {
         this._worldStateExtensions = worldStateExtensions
     }
 
-    get stats() {
-        return this._scenario.stats
+    get stats(): Stat<WorldState>[] {
+        return this._scenario.stats.map<Stat<WorldState>>((stat) => ({
+            ...stat,
+            getValue: ({ params }) => params.state[stat.id],
+        }))
     }
 
     /**
      * Get the initial state for a given scenario.
      *
      * @param scenario The scenario default data which holds all cards, events and similar
```

Comment:
Just noticed we have outdated docs here. Could you remove the `@param` line?

Reply:
This is a previous mistake not related to this PR. I'd rather fix it separately

Reply:
Fair enough :)